hugo/content/_index.html

```diff
@@ -2,35 +2,60 @@
 title: "Open Collaboration Tools"
 type: oct
 ---
-    <section class="bg-indigo-dark hero-section">
-        <div class="content">
-            <div class="hero-text fg-indigo-light">
-                <div class="hero-text-content">
-                    <h1 class="urbanist-bold">Collaborative Editing for your IDE or custom Editor</h1>
-                    <p class="urbanist-regular">Open Collaboration Tools is an set of open-source tools, libraries and extensions to allow real-time collaborative editing across devices and tools</p>         
-                    <p class="urbanist-regular">It's built to be easily extensible, to enable creating custom clients or modifying the server to suit your specific needs</p>
-                </div>
-            </div>
-            <div class="hero-image">
-                <image src="/assets/oct-image.png" alt="Collaborative Editing" />
+<section class="bg-indigo-dark hero-section">
```

Comment:
i would increase the bottom padding of this section. Looks a little off right now